### ISSUE TITLE
it doesn't work

### ISSUE THREAD

**Issue by ModMaker101** [RavinMaddHatter/Bedrock-Chunk-Boarders-Pack#19]
just doesn't work

**Comment by RavinMaddHatter** [comment]
It does... If you're in 3rd person... But you probably didn't bother to
watch the instructions

**Comment by Weltraumeule** [comment]
Hello, I'm on MC Bedrock 1.21.80 and today discovered your cool addon. Sadly it doesn't work anymore. Tested both with nautilus shell and also firework rocket. And I switched trough every view modes. 
EDIT: I'm using raytracing, maybe this causes the chunk borders grid to be invisible? 
EDIT2: I can see the chunk borders with the Bedrock-Technical-Resource-Pack

**Comment by RavinMaddHatter** [comment]
Well others say it is working just fine. You likely have other packs on
that are breaking it

**Comment by Weltraumeule** [comment]
You are right. The position between all other resorce packs is the key. At top Bedrock-Chunk-Boarders-Pack is working now! 
EDIT: No, I'm wrong. I had also the Bedrock-Technical-Resource-Pack enabled, therefore thought Bedrock-Chunk-Boarders-Pack was working. But it is ok, I will use Bedrock-Technical-Resource-Pack instead.

**Comment by ModMaker101** [comment]
That'll do it lmao

**Comment by Weltraumeule** [comment]
Btw. witj the Bedrock-Technical-Resource-Pack I can see the chunk borders also in first person view.

**Comment by RavinMaddHatter** [comment]
That was an update bud made. There is a video on it
